lib/tokens/token.js

```diff
@@ -40,7 +42,6 @@ module.exports = (log, config) => {
     KEYS.forEach(name => {
       this[name] = keys[name] && keys[name].toString('hex')
     })
-    this.algorithm = 'sha256'
```

Comment:
This throws in strict mode. `algorithm` is a defined property with a getter that returns `'sha256'` but no setter. This line was clobbering it. Presumably we prefer the getter to the raw property (for precisely this reason), hence I deleted this line.